frontend/src/components/BCDataGrid/BCGridBase.jsx

```diff
@@ -91,14 +91,6 @@ export const BCGridBase = forwardRef(
       clearFilters
     }))
 
-    const closeSelectsOnScroll = (e) => {
```

Comment:
This was causing lots of issues with Tab losing focus of cells, so lets just nuke it